### PR TITLE
227 1

### PR DESCRIPTION
**Pull request by ivicac**
master

### CODE REVIEW

File: client/src/pages/automation/project/Project.tsx [hunk 1]
```diff
@@ -187,22 +189,28 @@ const Project: React.FC = () => {
...
-                        filter={filter}
-                    />
+                    <>
+                        {components && flowControls && (
+                            <LeftSidebar
+                                data={{components, flowControls}}
+                                filter={filter}
+                            />
+                        )}
+                    </>
```
Comment (kresimir-coko) [inline client/src/pages/automation/project/Project.tsx]: Viška fragment, i dodati `!!` na `flowControls`

File: client/src/pages/automation/project/Project.tsx [hunk 2]
```diff
@@ -131,7 +133,7 @@ const Project: React.FC = () => {
                         <h1 className="mr-6 py-4 pr-4">{project?.name}</h1>
 
                         <div className="my-4 mx-2 flex rounded-md border border-gray-100 bg-white">
-                            {currentWorkflow && (
+                            {currentWorkflow && projectWorkflows && (
```
Comment (kresimir-coko) [inline client/src/pages/automation/project/Project.tsx]: `!!projectWorkflows`

File: client/src/pages/automation/project/Project.tsx [hunk 3]
```diff
@@ -187,22 +189,28 @@ const Project: React.FC = () => {
...
-                    flowControls={flowControls}
-                />
+                <>
+                    {components && flowControls && (
+                        <WorkflowEditor
+                            components={components}
+                            flowControls={flowControls}
+                        />
+                    )}
+                </>
```
Comment (kresimir-coko) [inline client/src/pages/automation/project/Project.tsx]: Viška fragment, i dodati `!!` na `flowControls`

File: client/src/pages/automation/project/Project.tsx (hunk 1 above)
Comment (ivicac) [reply client/src/pages/automation/project/Project.tsx]: Rijeseno

File: client/src/pages/automation/project/Project.tsx (hunk 3 above)
Comment (ivicac) [reply client/src/pages/automation/project/Project.tsx]: Rijeseno

File: client/src/pages/automation/project/Project.tsx (hunk 2 above)
Comment (ivicac) [reply client/src/pages/automation/project/Project.tsx]: Rijeseno

**Pull request merged**